comictalker/talkers/comicvine.py

```diff
@@ -0,0 +1,759 @@
+    id: int
+    name: str
+    site_detail_url: str
+    role: str
+
+
+class CVVolumeResults(TypedDict):
+    aliases: str
+    count_of_issues: int
+    description: str
+    id: int
+    image: CVImage
+    name: str
+    publisher: CVPublisher
+    start_year: str
+    resource_type: str
+
+
+class CVResult(TypedDict):
+    error: str
+    limit: int
+    offset: int
+    number_of_page_results: int
+    number_of_total_results: int
+    status_code: int
+    results: (CVIssueDetailResults | CVVolumeResults | list[CVVolumeResults] | list[CVIssueDetailResults])
+    version: str
+
+
+class CVVolumeFullResult(TypedDict):
+    characters: list[CVCredits]
+    locations: list[CVCredits]
+    people: list[CVPersonCredits]
+    site_detail_url: str
+    count_of_issues: int
+    description: str
+    id: int
+    name: str
+    publisher: CVPublisher
+    start_year: str
+    resource_type: str
+
+
+class CVIssueDetailResults(TypedDict, total=False):
+    aliases: str
+    api_detail_url: str
+    associated_images: list[CVAltImages]
+    character_credits: list[CVCredits]
+    character_died_in: None
+    concept_credits: list[CVCredits]
+    cover_date: str
+    date_added: str
+    date_last_updated: str
+    deck: None
+    description: str
+    first_appearance_characters: None
+    first_appearance_concepts: None
+    first_appearance_locations: None
+    first_appearance_objects: None
+    first_appearance_storyarcs: None
+    first_appearance_teams: None
+    has_staff_review: bool
+    id: int
+    image: CVImage
+    issue_number: str
+    location_credits: list[CVCredits]
+    name: str
+    object_credits: list[CVCredits]
+    person_credits: list[CVPersonCredits]
+    site_detail_url: str
+    store_date: str
+    story_arc_credits: list[CVCredits]
+    team_credits: list[CVCredits]
+    team_disbanded_in: None
+    volume: CVVolume
+
+
+CV_RATE_LIMIT_STATUS = 107
+
+
+class ComicVineTalker(ComicTalker):
+    def __init__(
+        self, api_url, api_key, series_match_thresh, remove_html_tables, use_series_start_as_volume, wait_on_ratelimit
+    ):
+        super().__init__()
+        self.source_details = SourceDetails(
+            name="Comic Vine",
+            ident="comicvine",
+            logo="comictalker/talkers/logos/comicvine.png",
+        )
+        # TODO Remove or leave in for future?
+        self.static_options = SourceStaticOptions(
+            logo_url="https://comicvine.gamespot.com/a/bundles/comicvinesite/images/logo.png",
+            website="https://comicvine.gamespot.com/",
+            has_issues=True,
+            has_alt_covers=True,
+            requires_apikey=True,
+            has_nsfw=False,
+            has_censored_covers=False,
+        )
+        # TODO Remove or leave in for future?
+        self.settings_options = {
+            "enabled": SourceSettingsOptions(
+                name="enabled", text="Enabled", help_text="", hidden=True, type=bool, value=True
+            ),
```

Comment:
Should I remove all of these or leave them in for reference?

Reply:
Lets remove them and we can take a fresh look at them in a later PR